lib/App/Sqitch/Engine/pg.pm

```diff
@@ -61,7 +61,7 @@ has _psql => (
             [ user   => $self->username ],
             [ dbname => $uri->dbname    ],
             [ host   => $uri->host      ],
-            [ port   => $uri->port      ],
+            [ port   => $uri->_port      ],
```

Comment:
It might be worth adding a comment that `_port` is in fact the documented way to get a port-sans-default from a `URI` (gross API).

Reply:
Good idea, avoid the issue in the future (hopefully). Done in 8f72ee6a.